javascript/javascript1/week4/class-exercises/fizzbuzz.js

```diff
@@ -0,0 +1,16 @@
+const start = 1;
+const end = 100;
+
+function fizzBuzz(numFizz, numBuzz) {
+  let output = [];
+  for (let i = start; i <= end; i++) {
+    if (i % numFizz === 0 && i % numBuzz === 0) output.push(`FizzBuzz`);
+    else if (i % numFizz === 0) output.push(`Fizz`);
+    else if (i % numBuzz === 0) output.push(`Buzz`);
+    else output.push(i);
+  }
+  return output.join('\n');
+}
+
+console.log(fizzBuzz(4, 12));
```

Comment:
With this combo you'd never get "Buzz", but that's nothing wrong with the code.

Reply:
You are right. I changed this test.